util/fibers/listener_interface.cc

```diff
@@ -110,6 +58,30 @@ struct ListenerInterface::TLConnList {
   }
 };
 
+void ListenerInterface::TLConnList::Link(Connection* c) {
+  DCHECK(!c->hook_.is_linked());
+  list.push_back(*c);
```

Comment:
the absence of underscores for members scares me now 😄

Reply:
yeah, need to switch to `this->` notation